src/app/lists/CommentBox.tsx

```diff
@@ -88,16 +88,16 @@ const CommentContainer: React.FC<CommentProps> = ({ originalComment }) => {
             setSuccessMessage("");
         } else {
             setErrorMessage("");
-            setResetComment(value);
             setSuccessMessage("Comment successfully updated.");
```

Comment:
do you need a success message if we are immediately reloading?

Reply:
Yes, it flashes up for a second before the reload happens!